catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -91,6 +93,32 @@
 
   private MetacardType nsiliMetacardType;
 
+  private static Map<String, String> typeConversionMap;
+
+  static {
+    typeConversionMap = new HashMap<>();
+    typeConversionMap.put("COLLECTION/EXPLOITATION PLAN", "Collection");
+    typeConversionMap.put("DOCUMENT", "Text");
+    typeConversionMap.put("GEOGRAPHIC AREA OF INTEREEST", "Dataset");
+    // typeConversionMap.put("GEOSPATIAL VECTOR", "");
+    // typeConversionMap.put("GMTI", "");
+    typeConversionMap.put("IMAGERY", "Image");
+    // typeConversionMap.put("INTELLIGENCE REQUIREMENT", "");
+    typeConversionMap.put("MESSAGE", "Text");
+    // typeConversionMap.put("OPERATIONAL ROLES", "");
+    // typeConversionMap.put("ORBAT", "");
+    // typeConversionMap.put("REPORT", "");
+    // typeConversionMap.put("RFI", "");
+    // typeConversionMap.put("SYSTEM ASSIGNMENTS", "");
+    // typeConversionMap.put("SYSTEM SPECIFICATIONS, "");
+    // typeConversionMap.put("SYSTEM DEPLOYMENT STATUS, "");
+    // typeConversionMap.put("TACTICAL SYMBOL", "");
+    // typeConversionMap.put("TDL DATA", "");
+    typeConversionMap.put("VIDEO", "Video");
+    typeConversionMap.put("CBRN", "Text");
+    typeConversionMap.put("ELECTRONIC ORDER OF BATTLE", "Interactive Resource");
```

Comment:
Should the commented out items be removed for a future ticket?

Reply:
Same comment as above. Future ticket with potentially additional logic to translate between non-complementary data types.

Reply:
https://codice.atlassian.net/browse/CAL-520 will address this